pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -131,6 +131,10 @@ type CertificateSpec struct {
 	// If provided, value must be between 2048 and 8192 inclusive when KeyAlgorithm is
 	// empty or is set to "rsa", and value must be one of (256, 384, 521) when
 	// KeyAlgorithm is set to "ecdsa".
+	// +kubebuilder:validation:ExclusiveMaxium=true
+	// +kubebuilder:validation:Maxium=8192
```

Comment:
typo: `maximum` - also, should 8192 be the exclusive maximum? I think that means that `8192` isn't a valid value.